media-api/test/lib/elasticsearch/QueryBuilderTest.scala

```diff
@@ -222,6 +230,14 @@ class QueryBuilderTest extends FunSpec with Matchers with ConditionFixtures {
     }
   }
 
+  describe("get advanced file metadata") {
+    it("should return the elasticsearchPath or the field parameter itself if the config doesn't exist"){
```

Comment:
This is testing two things. If you split it into two then you can more precisely describe the actual tests.